beets/ui/commands.py

```diff
@@ -787,7 +787,7 @@ def choose_item(self, task):
                 search_id = manual_id(True)
                 if search_id:
                     candidates, rec = autotag.tag_item(task.item,
-                                                       search_id=search_id)
+                                                       search_ids=[search_id])
```

Comment:
A related side effect (provided the current implementation of multiple IDs stays more or less the same) is that it would take almost no effort (mainly a matter of splitting `search_id`) to allow the user to specify several IDs during the import when selecting the `enter Id` prompt choice. Would that be a good idea?